lib/templates/bootstrap/textbox.js

```diff
@@ -1,5 +1,6 @@
 var React = require('react');
 var { View, Text, TextInput } = require('react-native');
+import TextField from 'react-native-md-textinput';
```

Comment:
actually I didn't mean to PR this commit. It's just for my project. My PR is related to next one, after this commit